exercises/practice/dnd-character/dnd_character_test.rb

```diff
@@ -0,0 +1,52 @@
+require 'minitest/autorun'
+require_relative 'dnd_character'
+
+module Helpers
+  BASE_HITPOINTS = 10
+  MODIFIER_TO_CONSTITUTION = {
```

Comment:
Constitution modifier to affect hitpoints, so this could be something that may be a different name.

Reply:
Gave a more specific name: `CONSTITUTION_MODIFIER_TO_CONSTITUTION_MAP`

Reply:
Not a great name.  But naming is hard.  And the `_MAP` is a "type indicator" in the name, which is not common in Ruby.

Let me look at this and see if I can come up with a better name.